### DISCUSSION TITLE
Plugin ID changed

### DISCUSSION THREAD

**Discussion by jfrad** [apohl79/audiogridder#1182] [Help]
Hi! here's my little issue, I apologise if this has been explained, I did search for it.

I just installed a new computer, mac studio, protools.
Upon opening older sessions from my previous computer the plugins don't load because the plugin id on the new computer is different.
Is there a way to fix this? To assign the same ID the the new install?

My concern is: what happens if the computer crashes and you need to wipe it? Will you lose all settings in your sessions?

**Comment by apohl79** [comment]
Did you install a new version of the plugin or changed the format?

The baseline of a plugin ID is generated by the plugin itself. AG did change its internal ID representation from 1.1 to 1.2 but has a backwards compatibility layer to load sessions stored with AG 1.1.
